src/internal_events/splunk_hec.rs

```diff
@@ -44,3 +45,26 @@ impl InternalEvent for SplunkEventEncodeError {
         );
     }
 }
+
+#[derive(Debug)]
+pub struct SplunkSourceMissingKeys {
```

Comment:
I expect to open another PR making `sourcetype` configurable as well which I _think_ should be a separate event?